corehq/apps/cloudcare/templates/formplayer/grid_view.html

```diff
@@ -111,10 +111,13 @@ <h3>{% trans "Settings" %}</h3>
   <div class="container">
     <div class="spacer"></div>
     <div class="col-xs-6 col-sm-4 col-xs-offset-3 col-sm-offset-4">
-      <div class="appicon appicon-default js-start-app" <% if (imageUrl) { %>style="background-image: url('<%= imageUrl %>');"<% } %>>
+      <div class="appicon appicon-default js-start-app">
         <% if (!imageUrl) { %>
           <i class="fcc fcc-flower appicon-icon" aria-hidden="true"></i>
         <% } %>
+        <% if (imageUrl) { %>
```

Comment:
Nitpick: not sure if you're intentionally using a second `if`, but `else` does work here, if you change the previous line to `<% } else { %>`

Reply:
Ah, I was not. good call. Fixed